examples/plot_simulate_evoked.py

```diff
@@ -56,5 +58,8 @@
 params.update({'sync_evinput': True})
 net_sync = Network(params)
 dpls_sync = simulate_dipole(net_sync, n_jobs=1, n_trials=1)
-dpls_sync[0].plot()
-net_sync.plot_input()
+if get_rank() == 0:
+    dpls_sync[0].plot()
+    net_sync.plot_input()
+
+shutdown()
```

Comment:
wrap this in a context manager? In the `__exit__` method of `net`?

Reply:
Unfortunately, that won't work. In order to get the MPI simulation to shutdown cleanly, h.quit() needs to be called which stops the nrniv process, and in turn stops the python script. Is there another way to automatically call h.quit() on exit from the python script?

Reply:
I'll think about a function to check if MPI is being used sp that this shutdown() command doesn't seem so random. It is necessary to call at the end of any example script.

if using_MPI():
  shutdown()

Reply:
I would make it all lower case and properly document the function

Reply:
no longer needed